paymentsheet/src/test/java/com/stripe/android/paymentsheet/ui/PaymentMethodUIScreenshotTest.kt

```diff
@@ -94,4 +93,76 @@ internal class PaymentMethodUIScreenshotTest {
             )
         }
     }
+
+    @Test
+    fun testSelectedStateForVerticalMode() {
+        paparazziRule.snapshot {
+            NewPaymentMethodRowButton(
+                isEnabled = true,
+                isSelected = true,
+                iconRes = R.drawable.stripe_ic_paymentsheet_pm_card,
+                iconUrl = null,
+                imageLoader = mock(),
+                title = "Card",
+                subTitle = null,
+                iconRequiresTinting = true,
+                onClick = {},
+                modifier = Modifier,
+            )
+        }
+    }
+
+    @Test
+    fun testUnselectedStateForVerticalMode() {
+        paparazziRule.snapshot {
+            NewPaymentMethodRowButton(
+                isEnabled = true,
+                isSelected = false,
+                iconRes = R.drawable.stripe_ic_paymentsheet_pm_card,
+                iconUrl = null,
+                imageLoader = mock(),
+                title = "Card",
+                subTitle = null,
+                iconRequiresTinting = true,
+                onClick = {},
+                modifier = Modifier,
+            )
+        }
+    }
+
+    @Test
+    fun testDisabledForVerticalMode() {
+        paparazziRule.snapshot {
+            NewPaymentMethodRowButton(
+                isEnabled = false,
+                isSelected = false,
+                iconRes = R.drawable.stripe_ic_paymentsheet_pm_card,
+                iconUrl = null,
+                imageLoader = mock(),
+                title = "Card",
+                subTitle = null,
+                iconRequiresTinting = true,
+                onClick = {},
+                modifier = Modifier,
+            )
+        }
+    }
+
+    @Test
+    fun testLongTitleForVerticalMode() {
+        paparazziRule.snapshot {
+            NewPaymentMethodRowButton(
+                isEnabled = true,
+                isSelected = false,
+                iconRes = R.drawable.stripe_ic_paymentsheet_pm_card,
```

Comment:
can you use a bank icon here instead of the card one? The screenshots are a bit confusing/unrealistic bc the title refers to a bank account but the icon is for a card